Popup/Popup.module.less

```diff
@@ -46,6 +47,12 @@
 			}
 		}
 
+		&.centeredContent {
+			text-align: center;
+			.buttons {
+				text-align: center;
+			}
+		}
```

Comment:
If you need to add some styles which are not related with skins, we recommend to define them out of the `applySkins`.
e.g>
```
.popup {
	&.centeredContent {
		text-align: center;
		.buttons {
			text-align: center;
		}
	}

	.applySkins({
		position: fixed;
		top: 50%;
```

If not, those styles will be generated to css as many as the number of the skins. So it is not efficient.

Reply:
Thank you for your feedback.
Should the other properties which do not have variables for values be defined outside the applySkins as well?
eg>		position: fixed;
		top: 50%;
		left: 50%;
		transform: translate(-50%,-50%);

Reply:
Yes, please. I missed that part. :)

Reply:
Could you please have another look now to see if I got it right? 
Thank you!

Reply:
Now it looks fine to me except the minor indentation part that I commented.
Thank you.